packages/client/wallets/aa/src/blockchain/token/Tokens.ts

```diff
@@ -1,18 +1,81 @@
-import { EVMBlockchain } from "@crossmint/common-sdk-base";
+import { BigNumber } from "ethers";
+
+import { EVMBlockchainIncludingTestnet, isEVMBlockchain } from "@crossmint/common-sdk-base";
 
 export interface EVMToken {
-    chain: EVMBlockchain;
+    chain: EVMBlockchainIncludingTestnet;
     contractAddress: string;
-    tokenId?: string;
 }
+
+export interface NFTEVMToken extends EVMToken {
+    tokenId: string;
+    type: NFT;
+}
+
+export interface SFTEVMToken extends EVMToken {
+    tokenId: string;
+    type: SFT;
+}
+
+export interface ERC2OEVMToken extends EVMToken {
+    type: FT;
```

Comment:
I think we're adding a layer of indirection we don't need– if these are strings, they're treated as consts / their own type in TS
```suggestion
    type: "nft";
}

export interface SFTEVMToken extends EVMToken {
    tokenId: string;
    type: "sft";
}

export interface ERC2OEVMToken extends EVMToken {
    type: "ft";
```